### ISSUE TITLE
Pressing the tab key does not focus the commit list on the History tab

### ISSUE THREAD

**Issue by jamesgecko** [desktop/desktop#6312]
## Description

I use keyboard shortcuts to navigate GitHub Desktop. When pressing tab on the History tab, the focus ring skips over the commit list.

## Version

* GitHub Desktop: 1.5.0

* Operating system: macOS Mojave 10.14

## Steps to Reproduce
Scenario: I want to scroll back through my commit history. I want to press tab a bunch so that the top commit in my list is blue, then use the up and down arrow keys to scroll.

I'll start in the `Changes` tab with the focus ring on the `Summary (required)` field. 

1. Press CMD+2 to change to the `History` tab.
2. Minor bonus bug: The `History` tab widget now has an underline on it. I assume that this means that it has focus. I press tab to change the selected UI element.
    - Expected behavior: the `Select Branch to Compare...` text box will be focused.
    - Actual behavior: Nothing on the screen changes.
3. I press tab again. This time the `Current Repository` dropdown gets a focus ring.
4. I press tab three more times to cycle the focus ring through `Current Branch`, `Publish branch`, and `History`. Nothing surprising.
5. I press tab to move the focus ring into the `Select Branch to Compare...` textbox. A list of branches appears.
6. I press tab to move the cursor from `Select Branch to Compare...` into the list of commits that was visible before the list of branches appeared in step 5.
    - Expected behavior: The focus ring selects the most recent commit in the list.
    - Actual behavior: The focus ring selects the top item in file changed column.

## Additional Information
The blue rectangle is the "focus ring" I'm talking about
<img width="731" alt="screen shot 2018-11-29 at 11 19 39 am" src="https://user-images.githubusercontent.com/44252/49235744-baee7a80-f3c8-11e8-9915-ae066251882d.png">

Here is the screen in step 5
<img width="671" alt="screen shot 2018-11-29 at 11 20 48 am" src="https://user-images.githubusercontent.com/44252/49235837-ee310980-f3c8-11e8-9a63-3b8cd2f69177.png">

What I want to happen in step 6
<img width="685" alt="screen shot 2018-11-29 at 11 24 04 am" src="https://user-images.githubusercontent.com/44252/49236116-6f889c00-f3c9-11e8-982e-a302db6cf8e2.png">

What actually happens in step 6
<img width="722" alt="screen shot 2018-11-29 at 11 21 00 am" src="https://user-images.githubusercontent.com/44252/49235859-fab56200-f3c8-11e8-899f-ba88a44e780f.png">

### Logs
N/A

**Comment by billygriffin** [comment]
@jamesgecko Thanks for the report. I was also able to reproduce this, and I'm going to label this as "accessibility." We've got some accessibility-focused work coming up so this might be a good fit. 😄

**Comment by tierninho** [comment]
@jamesgecko There is a hacky workaround for this until we fix it. 

Click on History tab, then tab twice so the selector is on the diff like your last screenshot. Now `shift + tab` and you go back to the commit list.

**Comment by tierninho** [comment]
I'd say this has been inadvertently fixed as tabbing is now allowing the user to reach the commit history successfully. Step 3 is wedged between, thus looking like it doesn't work. 

Tab order is:
1) History
2) Compare filter
3) **Opening of commit history** (not supposed to highlight exact commit at this step)
4) Focus on current commit
5) Focus on current diff

Step 3
<img width="618" alt="Screen Shot 2019-05-07 at 8 08 15 AM" src="https://user-images.githubusercontent.com/14828183/57322579-90070180-709f-11e9-856f-c22c584ade81.png">

Step 4
<img width="596" alt="Screen Shot 2019-05-07 at 8 08 26 AM" src="https://user-images.githubusercontent.com/14828183/57322578-8f6e6b00-709f-11e9-9cdb-52af46d5cd70.png">

Step 5
<img width="608" alt="Screen Shot 2019-05-07 at 8 08 32 AM" src="https://user-images.githubusercontent.com/14828183/57322577-8f6e6b00-709f-11e9-8bb8-a126628d46c3.png">
